config/ember-try.js

```diff
@@ -7,46 +7,53 @@ module.exports = {
     {
       name: 'ember-1.10',
       dependencies: {
-        'ember': '~1.10.0'
+        ember: '~1.10.0'
       }
     },
     {
       name: 'ember-1.11',
       dependencies: {
-        'ember': '~1.11.3'
+        ember: '~1.11.3'
       }
     },
     {
       name: 'ember-1.12',
       dependencies: {
-        'ember': '~1.12.0'
+        ember: '~1.12.0'
+      }
+    },
+    {
+      name: 'ember-1.13',
+      dependencies: {
+        ember: '~1.13.0'
       }
     },
     {
       name: 'ember-release',
       dependencies: {
-        'ember': 'components/ember#release'
+        ember: 'components/ember#release'
```

Comment:
I'd add an `'ember-data': '~2.0.0'` dependency to everything from release through canary